src/server/actions/Organizations.ts

```diff
@@ -19,3 +19,58 @@ export async function softDeleteOrganization(
 
   return res;
 }
+
+/**
+ * update existing organization information
+ * @param organizationId The Id of the organization to be updated
+ * @param updatedData The data to be updated in database
+ * @returns The organization in the database before soft-deletion, or null
+ */
+export async function updateOrganization(
+  organizationId: string,
+  updatedData: object
```

Comment:
this should be of type UpdateOrganizationRequest. This will prevent updating the `softDelete` field after we have validated the request